bootstrap.sh

```diff
@@ -8,7 +8,9 @@ apt install -y python3.7 python3-pip postgresql postgresql-contrib
 
 echo "installing dependencies"
 pip3 install --upgrade pip
-pip3 install -r /vagrant/requirements.txt
+pip3 install pipenv
+export PIPENV_VENV_IN_PROJECT=1
```

Comment:
I don't think you really need this - if you do, add a comment why, since this is not the default way of using pipenv.

Reply:
When I removed this line, the virtual environment is always created under root user. After I try to SSH into the machine the logged in user is vagrant and so I couldn't access the virtual environment.